### PR TITLE
Page/home

### PR DESCRIPTION
**Pull request by arthurfanti**
### WHAT
Home Page

### CODE REVIEW

**Comment by arthurfanti** [comment]
[Home page](https://trello.com/c/vnKXfYip/3-home-page)

File: src/_pages/home.html [hunk 1]
```diff
@@ -37,50 +37,121 @@
...
               <div class="row">
-                <div class="col-sm-9">
+                <div class="col-sm-6 col-sm-push-3 text-center">
+                  <h3>Sharing Documents & Links</h3>
+                  <p>
+                    <cite>"We use Skore as a "content hub" for everything sales enablement and customer messaging related. Our content can come from anywhere: PDFs, Google Drive, links to websites, etc. Skore is the hub on the spoke of the wheel that points to all the content."</cite>
+                  </p>
+
+                  <div class="user-avatar">
+                    <img src="https://randomuser.me/api/portraits/lego/6.jpg" alt="">
```
Comment (vinicius-sabadim) [inline src/_pages/home.html]: Don't forget to put the `alt` when this is finished :D

File: src/assets/scss/base/_helpers.scss [hunk 2]
```diff
@@ -111,36 +111,57 @@
...
 }
-.light-gray {
-  color: $light-gray;
+.black {
+  color: $black;
 }
-.gray {
-  color: $gray;
+.cool-gray {
+  color: $cool-gray;
```
Comment (vinicius-sabadim) [inline src/assets/scss/base/_helpers.scss]: What's this? haha

File: src/assets/scss/skore/home/_header.scss [hunk 3]
```diff
@@ -5,9 +5,9 @@
   background-position-y: 120px;
   background-repeat: no-repeat;
   color: $white;
-  margin-top: -52px;
-  min-height: 340px;
-  padding: 52px 0 20%;
+  margin-top: em(-84px);
```
Comment (vinicius-sabadim) [inline src/assets/scss/skore/home/_header.scss]: :x

File: src/assets/scss/base/_helpers.scss (hunk 2 above)
Comment (arthurfanti) [inline src/assets/scss/base/_helpers.scss]: @vinicius-sabadim coisas de alemão!

File: src/assets/scss/skore/home/_header.scss (hunk 3 above)
Comment (arthurfanti) [inline src/assets/scss/skore/home/_header.scss]: @vinicius-sabadim its not a hack, i swear!

**Comment by femontanha** [comment]
[Top Bar](https://trello.com/c/sJcb5oBe/9-top-bar)

**Pull request merged**